index.html

```diff
@@ -790,12 +790,7 @@ <h3>Presentations</h3>
 
         <p>
 A <a>verifiable presentation</a> expresses data from one or more
-<a>verifiable credentials</a>, and is packaged in such a way that the
-authorship of the data is <a>verifiable</a>. If <a>verifiable credentials</a>
-are presented directly, they become <a>verifiable presentations</a>. Data
```

Comment:
I like some of this text, don't think it should all be removed.

Reply:
I'm open to suggestions. My attempt to adjust this section did not find consensus.

Reply:
can we keep 

```
<a>verifiable credentials</a>, and is packaged in such a way that the
authorship of the data is <a>verifiable</a>. If <a>verifiable credentials</a>
are presented directly, they become <a>verifiable presentations</a>.
```

Reply:
Now that we've introduced media types, I don't think it is true that  a verifiable credential (vc+ld+json) becomes a verifiable presentation (vp+ld+json).

What about a sentence like: "It is also possible to present verifiable credentials directly."

Reply:
sure I think that's a good sentence

Reply:
Re-worded in [12f46a4](https://github.com/w3c/vc-data-model/pull/1181/commits/12f46a47beeeeb949b9ed15b6553c06338a6ec87)